hosted-cluster-config-operator/controllers/cmca/setup.go

```diff
@@ -24,16 +28,25 @@ func Setup(cfg *operator.HostedClusterConfigOperatorConfig) error {
 }
 
 func setupConfigMapObserver(cfg *operator.HostedClusterConfigOperatorConfig) error {
-	informerFactory := cfg.TargetKubeInformersForNamespace(ManagedConfigNamespace)
+	targetKubeClient, err := kubeclient.NewForConfig(cfg.TargetConfig)
```

Comment:
I think the reason for the `TagetKubeInformersForNamespace` to exist was to share informers between controllers which we now don't do anymore, but IMHO we should continue doing that or not?

Reply:
If we consolidate these controllers, we won't have a need for it

Reply:
hmm ok. There is at least one controller (cmca) that reads data from the guest cluster that is created by a clusteroperator and which the cmca conroller never updates, which means the labelselector-on-cache approach won't work there, it needs its own cache. But i suppose we can defer that issue for later.